apps/web/src/components/Composer/Actions/CollectSettings/SplitConfig.tsx

```diff
@@ -113,53 +114,60 @@ const SplitConfig: FC<SplitConfigProps> = ({
       />
       {hasRecipients ? (
         <div className="space-y-3 pt-4">
-          <div className="space-y-2">
-            {recipients.map((recipient, index) => (
-              <div key={index} className="flex items-center space-x-2 text-sm">
-                <Input
-                  placeholder="0x3A5bd...5e3 or wagmi.lens"
-                  value={recipient.recipient}
-                  disabled={loading}
-                  error={
-                    recipient.recipient.length > 0 &&
-                    !isValidEthAddress(recipient.recipient)
-                  }
-                  onChange={(event) =>
-                    onChangeRecipientOrSplit(
-                      index,
-                      event.target.value,
-                      'recipient'
-                    )
-                  }
-                />
-                <div className="w-1/3">
-                  <Input
-                    type="number"
-                    placeholder="5"
-                    min="1"
-                    max="100"
-                    value={recipient.split}
-                    iconRight="%"
-                    onChange={(event) =>
-                      onChangeRecipientOrSplit(
-                        index,
-                        event.target.value,
-                        'split'
-                      )
-                    }
-                  />
-                </div>
-                <button
-                  onClick={() => {
-                    setCollectType({
-                      recipients: recipients.filter((_, i) => i !== index)
-                    });
-                  }}
-                >
-                  <XCircleIcon className="h-5 w-5 text-red-500" />
-                </button>
-              </div>
-            ))}
+          <div className="[&>*]:space-y-2">
+            {recipients ? (
```

Comment:
I don't think you'd need this condition at all. We already check if recipients are there or not in line number `115`

Reply:
pushed